src/main/java/com/billennium/bcvsaver/repository/CvRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.billennium.bcvsaver.repository;
+
+import com.billennium.bcvsaver.entity.Cv;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CvRepository extends JpaRepository<Cv, Long> {
```

Comment:
You could add some unit tests to check basic functionalities using in-memory-database like h2